File: beginner/1094.py
n = int(input())
i = 0
cobaias = 0
coelhos = 0
ratos = 0
sapos = 0

while i < n:
	aux1, aux2 = input().split(" ")
	aux1 = int(aux1)
	cobaias += aux1
	if aux2 == "C":
		coelhos += aux1
	if aux2 == "R":
		ratos += aux1
	if aux2 == "S":
		sapos += aux1
	i += 1
	
print(f"Total: {cobaias} cobaias")
print(f"Total de coelhos: {coelhos}")
print(f"Total de ratos: {ratos}")
print(f"Total de sapos: {sapos}")
print(f"Percentual de coelhos: {((coelhos/cobaias) * 100):.2f} %")
print(f"Percentual de ratos: {((ratos/cobaias) * 100):.2f} %")
print(f"Percentual de sapos: {((sapos/cobaias) * 100):.2f} %")

File: beginner/1094.c
# include <stdio.h>

int main(){
	float n, i, cobaias, coelhos, ratos, sapos, auxInt;
	n = i = cobaias = coelhos = ratos = sapos = auxInt = 0;
	char auxChar;
	
	scanf("%f", &n);

	for( i = 0 ; i < n ; ++i ){
		scanf("%f %c", &auxInt, &auxChar);
		cobaias += auxInt;
		if( auxChar == 'C'){
			coelhos += auxInt;
		}
		if( auxChar == 'R'){
			ratos += auxInt;
		}
		if( auxChar == 'S'){
			sapos += auxInt;
		}
	}
	
	printf("Total: %.0f cobaias\n", cobaias);
	printf("Total de coelhos: %.0f\n", coelhos);
	printf("Total de ratos: %.0f\n", ratos);
	printf("Total de sapos: %.0f\n", sapos);
	printf("Percentual de coelhos: %.2f %%\n", (coelhos/cobaias) * 100.0 );
	printf("Percentual de ratos: %.2f %%\n", (ratos/cobaias) * 100.0 );
	printf("Percentual de sapos: %.2f %%\n", (sapos/cobaias) * 100.0 );

	return 0;
}
